packages/vitest/coverage.d.ts

```diff
@@ -0,0 +1 @@
+export * from './dist/coverage.js'
```

Comment:
`vitest/coverage` entry point should also be added to `tsconfig` as `vitest/src/coverage` otherwise it includes `dist` folder

Reply:
Updated the root `tsconfig.json`. Though I'm not exactly sure why that was needed, as the typings were already working in `test/coverage-test` and locally linked project.